scripts/wasmd/init.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+command -v shellcheck > /dev/null && shellcheck "$0"
+
+echo "Waiting for blockchain and REST server to be available ..."
+timeout 60 bash -c "until curl -s http://localhost:1319/node_info > /dev/null; do sleep 0.5; done"
+# The chain is unreliable in the first second of its existence (https://gist.github.com/webmaster128/8175692d4af5e6c572fddda7a9ef437c)
+sleep 1
+echo "Waiting for height to be > 1 ..."
+sleep 10
+# timeout 20 bash -c "until test $(curl -s localhost:1319/blocks/latest | jq -r '.block.header.height // 0') -gt 1; do sleep 0.5; done"
```

Comment:
@webmaster128 I think I'm doing something stupid here, but I can't get this line to work with the `timeout`.

Reply:
Hmm, `[`, the nice syntax for `test` always needs a space before the end. Maybe an issue here? 

```suggestion
# timeout 20 bash -c "until [ $(curl -s localhost:1319/blocks/latest | jq -r '.block.header.height // 0') -gt 1 ]; do sleep 0.5; done"
```

Reply:
I tried that syntax before `test`.

Reply:
Ahh, `$(curl -s localhost:1319/blocks/latest | jq -r '.block.header.height // 0')` is executed only once and then the following runs:

```
timeout 20 bash -c "until test 1 -gt 1; do sleep 0.5; done"
```

Reply:
Found a way to fix the script. I just wonder why we need min height 2 instead of min height 1?